apps/pg_subscriptor/lib/handler.ex

```diff
@@ -106,6 +110,30 @@ defmodule PgHandler do
     message
   end
 
+  defp handle_update!(body) do
+    Logger.info("Got UPDATE msg")
+    <<_relation_oid::32, rest::binary>> = body
+
+    {tuple_type, tuple_data, rest} =
+      case rest do
+        <<tuple_type::bitstring-size(8), rest::binary>> when tuple_type in ["O", "K"] ->
```

Comment:
> Changed `tuple_type` to bitstring to help elixir show that we expect
character.

What for? We are matching it right away, and if it is not one of O/K, we just return an error. We do not really care that it is a character, do we?

Reply:
I believe you can use `<<tuple_type>>`. I used something like that when debugging.